packages/interceptors-opentelemetry/package.json

```diff
@@ -33,6 +33,20 @@
     "@temporalio/worker": "file:../worker",
     "@temporalio/workflow": "file:../workflow"
   },
+  "peerDependenciesMeta": {
+    "@temporalio/activity": {
+      "optional": true
+    },
+    "@temporalio/client": {
+      "optional": true
+    },
+    "@temporalio/worker": {
+      "optional": true
+    },
```

Comment:
These are only used for their types

Reply:
Oh? In that case, can't we make them dev dependencies, instead of optional peer?

Reply:
Technically, the version will be guaranteed by the dependency on common anyway, but fair.

Reply:
Ah, you're right! I didn't think about that